tests/utils.go

```diff
@@ -674,6 +677,50 @@ func NewRandomVMWithPVC(claimName string) *v1.VirtualMachine {
 	return vm
 }
 
+func NewRandomVMWithTwoPVC(claimName string) *v1.VirtualMachine {
+	vm := NewRandomVM()
```

Comment:
You can use `NewRandomVMWithPVC` and add to it additional disk

Reply:
Done, thanks for pointing it out.